tests/test_group_layer_widget.py

```diff
@@ -1,18 +1,49 @@
 import pytest
 from napari_experimental._widget import GroupLayerWidget
 from napari_experimental.group_layer import GroupLayer, GroupLayerNode
+from napari_experimental.group_layer_actions import GroupLayerActions
 from napari_experimental.group_layer_delegate import GroupLayerDelegate
-from qtpy.QtCore import QPoint
+from qtpy.QtCore import QPoint, Qt
 from qtpy.QtWidgets import QWidget
 
 
 @pytest.fixture()
 def group_layer_widget(make_napari_viewer, blobs):
+    """Group layer widget with one image layer"""
     viewer = make_napari_viewer()
     viewer.add_image(blobs)
     return GroupLayerWidget(viewer)
 
 
+@pytest.fixture()
+def group_layer_widget_with_nested_groups(
```

Comment:
We have a `fixtures` folder for fixtures that we think we'll be recycling. The widget being here is all good, but the underlying `GroupLayer` instance that you're creating could become a fixture inside `fixtures/confest_group_layers.py`? There are also some existing fixtures in there with a nested group structure.

Reply:
Happy to re-use the existing fixtures with a nested group structure. Is there any way to directly set the group layers on the widget? As it's constructed from the viewer layers, I guess I'd always have to add any group layer as a sub-group of the existing root node?

Reply:
Don't quote me on this, but I'd of thought you should be able to instantiate the widget with an empty layerlist, then reassign the group_layers property to the testing fixture.

```python
def a_test(group_layers_widget, nested_layer_group):
    group_layers_widget.group_layers = nested_layer_group
```

Though on reflection, I would guess that might lead to issues when sync-ing layers back to the main view. You might need to

```python
def a_test(make_napari_viewer, group_layers_widget, nested_group_layer):
   viewer = make_napari_viewer()
   for node in nested_group_layer:
      if not node.is_group():
          viewer.add_points(node.layer)

   group_layers_widget.group_layers = nested_layer_group
```

instead. Though if there's no easy way to refactor, feel free to ignore me here - it's not a major issue, just thought it would be good to re-use the same nested structure across all our tests so we don't have to constantly look up different layouts.

Reply:
Ok - sounds good. I think I'll leave as-is for now - then if I find some time next week I can always refactor these tests.